src/main/java/it/gov/pagopa/cgn/portal/service/DiscountService.java

```diff
@@ -41,11 +44,8 @@ public DiscountEntity createDiscount(String agreementId, DiscountEntity discount
     public DiscountEntity updateDiscount(String agreementId, Long discountId, DiscountEntity discountEntity) {
         agreementServiceLight.findById(agreementId);
```

Comment:
What's the purpose of this query?

Reply:
To check if the agreement exists. Method findById will throw an exception if Agreement is not found

Reply:
👍 Can you provide a little comment about this behaviour ?

Reply:
Done, jus pushed.